js/views/modals/editListing/EditListing.js

```diff
@@ -18,6 +20,29 @@ import app from '../../../app';
 import BaseModal from '../BaseModal';
 import ShippingOption from './ShippingOption';
 
+$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
+
+function installHTMLSanitizationWatcher() {
```

Comment:
Is it really necessary to sanitize the HTML as the user is typing (or 2 seconds after)?

One problem is that when you update the field like that, the cursor position is lost, which is annoying. Let's say I was typing and took a two second or more break to read what I typed. Now, suddenly the cursor position jumps somewhere else when the text is sanitized.

The second issue is that, IMHO, this adds a whole lot of complexity to the code (this is not an easy function to follow) with a very low ROI.

I would opt for just initializing the plugin on render (`this$('#editListingDescription').trumbowyg();`) and just sanitizing on save.